src/overlays/actors/ovl_Bg_Hidan_Fslift/z_bg_hidan_fslift.c

```diff
@@ -72,69 +72,69 @@ void BgHidanFslift_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_80886FCC(BgHidanFslift* this, PlayState* play) {
-    s32 heightBool;
+void BgHidanFslift_WaitForPlayerOnTop(BgHidanFslift* this, PlayState* play) {
+    s32 nearHomePos;
 
     if (this->timer) {
         this->timer--;
     }
 
     if (this->timer == 0) {
-        heightBool = false;
+        nearHomePos = false;
         if ((this->dyna.actor.world.pos.y - this->dyna.actor.home.pos.y) < 0.5f) {
-            heightBool = true;
+            nearHomePos = true;
         }
-        if (DynaPolyActor_IsPlayerAbove(&this->dyna) && heightBool) {
-            this->actionFunc = func_808870D8;
-        } else if (!heightBool) {
-            this->actionFunc = func_8088706C;
+        if (DynaPolyActor_IsPlayerAbove(&this->dyna) && nearHomePos) {
+            this->actionFunc = BgHidanFslift_Ascend;
+        } else if (!nearHomePos) {
+            this->actionFunc = BgHidanFslift_Descend;
         }
     }
 }
 
-void func_8088706C(BgHidanFslift* this, PlayState* play) {
+void BgHidanFslift_Descend(BgHidanFslift* this, PlayState* play) {
     if (Math_StepToF(&this->dyna.actor.world.pos.y, this->dyna.actor.home.pos.y, 4.0f)) {
         Actor_PlaySfx(&this->dyna.actor, NA_SE_EV_BLOCK_BOUND);
-        func_80886FB4(this);
+        BgHidanFslift_SetupWaitForPlayerOnTop(this);
     } else {
         func_8002F974(&this->dyna.actor, NA_SE_EV_ELEVATOR_MOVE3 - SFX_FLAG);
     }
-    func_80886F24(this);
+    BgHidanFslift_CopyPosToHsblock(this);
 }
 
-void func_808870D8(BgHidanFslift* this, PlayState* play) {
+void BgHidanFslift_Ascend(BgHidanFslift* this, PlayState* play) {
     if (DynaPolyActor_IsPlayerAbove(&this->dyna)) {
         if (Math_StepToF(&this->dyna.actor.world.pos.y, this->dyna.actor.home.pos.y + 790.0f, 4.0f)) {
             Actor_PlaySfx(&this->dyna.actor, NA_SE_EV_BLOCK_BOUND);
-            func_80886FB4(this);
+            BgHidanFslift_SetupWaitForPlayerOnTop(this);
         } else {
             func_8002F974(&this->dyna.actor, NA_SE_EV_ELEVATOR_MOVE3 - SFX_FLAG);
         }
     } else {
-        func_80886FB4(this);
+        BgHidanFslift_SetupWaitForPlayerOnTop(this);
     }
-    func_80886F24(this);
+    BgHidanFslift_CopyPosToHsblock(this);
 }
 
 void BgHidanFslift_Update(Actor* thisx, PlayState* play) {
     BgHidanFslift* this = (BgHidanFslift*)thisx;
 
     this->actionFunc(this, play);
     if (DynaPolyActor_IsPlayerOnTop(&this->dyna)) {
-        if (this->unk_16A == 0) {
-            this->unk_16A = 3;
+        if (this->cameraSetting == CAM_SET_NONE) {
+            this->cameraSetting = CAM_SET_DUNGEON0;
         }
         Camera_ChangeSetting(play->cameraPtrs[CAM_ID_MAIN], CAM_SET_ELEVATOR_PLATFORM);
     } else if (!DynaPolyActor_IsPlayerOnTop(&this->dyna)) {
-        if (this->unk_16A != 0) {
+        if (this->cameraSetting != CAM_SET_NONE) {
             Camera_ChangeSetting(play->cameraPtrs[CAM_ID_MAIN], CAM_SET_DUNGEON0);
         }
-        this->unk_16A = 0;
+        this->cameraSetting = CAM_SET_NONE;
```

Comment:
yeah this is a bit strange. It does seem like they set the camera setting to the variable but then forgot to use the variable in the function call.

I guess its okay, but could use a comment explaining the speculative nature of the name, probably right above the function call where it would have been used

Reply:
Sounds good, added a comment to clarify if we want to keep the name - can wait for some others to weigh in for opinions as well.

Reply:
Worth noting this same pattern is present in other fire temple actors: `z_bg_hidan_syoku.c` and `z_bg_hidan_rock.c`. I think it's also fine